blueprints/load-scheduling/average-latency/policy.libsonnet

```diff
@@ -3,7 +3,13 @@ local commonPolicyFn = import '../common-aimd/policy.libsonnet';
 local config = import './config.libsonnet';
 
 function(cfg, params={}, metadata={}) {
-  local updatedConfig = config + cfg,
+  local updatedConfig = config + cfg + {
+    policy+: {
+      service_protection_core+: {
+        overload_condition: 'gt',
```

Comment:
Shouldn’t be part of service protection core. This should be part of base blueprint just like PromQL query